src/mutmap/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+find_package(Rscript)
+
+set(MUTMAP_DIR ${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBEXECDIR}/mutmap)
+
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/mutmap.sh.in"
+  "${CMAKE_CURRENT_BINARY_DIR}/mutmap.sh" @ONLY)
+
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/dist/build.R"
+  "${CMAKE_CURRENT_BINARY_DIR}/build.R" @ONLY)
+
+configure_file("${CMAKE_CURRENT_SOURCE_DIR}/dist/bundle.js"
+  "${CMAKE_CURRENT_BINARY_DIR}/bundle.js" @ONLY)
+
```

Comment:
Sorry I was assuming people compiling would be considered developers, but I guess most of our users compile from source. I agree a dependency on Node should be avoided, even though the visualization component will be optional.

I don't think the bundling will be too bad. I already have everything modularized I just need to convert the syntax to a vanilla js module pattern instead of the CommonJS semantics node uses.